crypto/fipsmodule/rand/rand.c

```diff
@@ -316,8 +320,11 @@ static void rand_get_seed(struct rand_thread_state *state,
     BORINGSSL_FIPS_abort();
   }
 
-  OPENSSL_STATIC_ASSERT(sizeof(entropy) % CRNGT_BLOCK_SIZE == 0, _)
-  for (size_t i = CRNGT_BLOCK_SIZE; i < sizeof(entropy);
+  // |entropy_len_including_overread| is one of
```

Comment:
I don't see in this function a check that verifies that `entropy_len` is one of the two possible values we expect. Is there a reason we don't have to check this?

Reply:
My rational was that it was only being called from `RAND_bytes_with_additional_data` which would only use valid arguments. I could add a
```
entropy_len < CTR_DRBG_MAX_ENTROPY_LEN
```
somewhere

Reply:
ok, since the function is `static` and called only by functions in this file which call it only with the correct lengths I think it's ok to go without a check. So resolving this comment as well